### PR TITLE
bundle for ES6 and ship types

### PR DESCRIPTION
**Pull request by brad-decker**
I tried to use React to build a simple example of using React with this library. To streamline my process I used Next (and then create-react-app) and both failed to bundle the package without ejecting from the build systems. Looking into it I saw that we have src/index.ts but we never generate src/index.js which was linked in the 'module' key of package.json. I also realized we'd want to have types. I found https://github.com/ezolenko/rollup-plugin-typescript2 during the process and it respects `declaration: true` when bundling.

### CODE REVIEW

**Comment by Gudahtt** [comment]
Does this problem exist with the published version of this package? The TypeScript conversion has yet to be published.

**Comment by brad-decker** [comment]
@Gudahtt I didn't yarn link to check the current repo code before I tried to fix it myself. The bug ONLY exists in the currently published version... if I switch to master and build it works... but I believe it's using the cjs files even when i'm using import syntax.

**Comment by brad-decker** [comment]
This does result in an es bundled version being found and used.

**Comment by Gudahtt** [comment]
> I believe it's using the cjs files even when i'm using import syntax.

Huh. That's strange. The `module` field (non-standard but supported by most bundlers) is supposed to be preferred over `main` I thought.

**Comment by brad-decker** [comment]
module points to a “index.js” that doesn’t appear to exist

**Comment by Gudahtt** [comment]
Sorry, I meant in the current published version (v0.3.0). It does exist ithere ([here](https://github.com/MetaMask/metamask-onboarding/blob/v0.3.0/src/index.js))

I tried getting it to work using `create-react-app` and it seemed to work.

`master` does indeed seem to be broken at the moment.

**Comment by Gudahtt** [comment]
I think I see the problem you originally encountered. NextJS fails for me as well, seemingly because they prefer `main` over `module` (unlike Webpack, which is why `create-react-app` works).

This is failing because I imported `'bowser/src/bowser' instead of `bowser`, because it was the only way to get the ES6 version of `bowser` (they didn't use a `module` field at the time). This is still used by the CommonJS bundle, which it of course shouldn't be.

File: package.json [hunk 1]
```diff
@@ -24,7 +25,7 @@
   "scripts": {
     "prepublishOnly": "npm run build",
     "lint": "eslint --ext .ts,.js .",
-    "build": "rollup --config && rollup --config rollup.config.standalone.js"
```
Comment (Gudahtt) [inline package.json:27]: Nit: It looks like the file `rollup.config.standalone.js` still exists, despite having been merged with `rollup.config.js`

**Comment by brad-decker** [comment]
@Gudahtt removed the excess file.

**Pull request merged**